### ISSUE TITLE
Add keyboard shortcut for "markFileReviewedAndAdvance"

### ISSUE THREAD

**Issue by devjgm** [Reviewable/Reviewable#611]
The default keybindings at https://reviewable.io/bindings.json have the following entry:

```
...
[null, "Mark file as reviewed and advance", "markFileReviewedAndAdvance()"],
...
```
This function appears to correspond with the "Mark reviewed and go to next file" click target in the UI. This is a common operation that I think should have a default keybinding.

I will propose the keybinding `"m"` for this because it appears to be available and seems to mneumonicly match "[m]arkFileReviewedAndAdvance".

**Comment by pkaminski** [comment]
I agree this feature deserves a default binding, but there's too many bindings as it is and I'll be revamping navigation functions altogether with the move to a sidebar.  I'll make sure mark-and-advance gets a binding at that time (possibly indirect, via a generic "next" function), but for now it should be easy enough to add your own custom binding.
